tests/console/hostname.pm

```diff
@@ -26,10 +26,14 @@ use strict;
 use warnings;
 use testapi;
 use utils;
+use version_utils "is_sle";
 
 sub run {
     select_console 'root-console';
 
+    # Since 15SP1 DHCLIENT_SET_HOSTNAME must be disable bsc#1153402
```

Comment:
Why you don't put this at the beginning of `set_hostname` function in `lib/utils.pm`?

Reply:
I did it at the beginning and I hesitated, maybe impact is less like that but I can move it to `lib/utils.pm` if it's more relevant to you.

Reply:
I think so, otherwise each time someone wants to call `set_hostname` he will have to do the same.

@alvarocarvajald WDYT?

Reply:
IMO this goes better here, but it could also be added to the documentation of `set_hostname`

Reply:
Ok.